### PR TITLE
[ui/ux] (Action Recorder) Document the copy as code button

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR documents the new **Copy as code** button in the Action Recorder. It gives the usage scenarios for why you'd use the button.

### CODE REVIEW

File: docs/develop/best-practices.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 ms.localizationpriority: medium
 ---
 
 # Best practices in Office Scripts
 
 These patterns and practices are designed to help your scripts run successfully every time. Use them to avoid common pitfalls as you start automating your Excel workflow.
 
+## Use the Action Recorder to learn new features
+
+Excel does a lot of things. Most of them can be scripted. The Action Recorder records your Excel actions and translates them into code. This is the easiest way to learn about how different features work Office Scripts. If you need code for a specific action, switch to the Action Recorder, perform the actions, select **Copy as code**, and paste the resulting code into your script.
```
Comment (alison-mk) [inline docs/develop/best-practices.md:14]: This sentence is missing a word. Perhaps: 
```suggestion
Excel does a lot of things. Most of them can be scripted. The Action Recorder records your Excel actions and translates them into code. This is the easiest way to learn about how different features work with Office Scripts. If you need code for a specific action, switch to the Action Recorder, perform the actions, select **Copy as code**, and paste the resulting code into your script.
```

File: docs/overview/excel.md [hunk 2]
```diff
@@ -54,9 +54,11 @@ The Action Recorder records actions you take in Excel and saves them as a script
 
 All scripts recorded with the Action Recorder can be edited through the Code Editor. This lets you tweak and customize the script to better suit your exact needs. You can also add logic and functionality that is not directly accessible through the Excel UI, such as conditional statements (if/else) and loops.
 
-One easy way to start learning the capabilities of Office Scripts is to record scripts in Excel on the web and view the resulting code. Another option is to follow our [tutorials](../tutorials/excel-tutorial.md) to learn in a more guided and structured way.
+> [!TIP]
+> The Action Recorder has a **Copy as code** button to actions into script code without saving the entire script.
```
Comment (alison-mk) [inline docs/overview/excel.md:58]: Missing a word. Perhaps: 
```suggestion
> The Action Recorder has a **Copy as code** button to record actions into script code without saving the entire script.
```

File: docs/overview/excel.md [hunk 3]
```diff
@@ -54,9 +54,11 @@ The Action Recorder records actions you take in Excel and saves them as a script
...
+> :::image type="content" source="../images/action-recorder-copy-code.png" alt-text="The Action Recorder task pane with the 'Copy as code' button highlighted.":::
```
Comment (alison-mk) [inline docs/overview/excel.md:59]: This image displays on the same line as the text in the script, creating a strange visual experience. Can we force a line break in the !TIP?

**Review by alison-mk — CHANGES_REQUESTED** [review]
My only blocking comment is about the strange appearance of the image inside the !TIP. Also left a couple comments suggesting fixes for typos.

File: docs/overview/excel.md (hunk 3 above)
Comment (AlexJerabek) [inline docs/overview/excel.md:59]:
```suggestion
>
> :::image type="content" source="../images/action-recorder-copy-code.png" alt-text="The Action Recorder task pane with the 'Copy as code' button highlighted.":::
```

Comment (AlexJerabek) [reply docs/overview/excel.md:59]: It's unfortunate that displays correctly in the VSCode preview. Good to know there's a difference!

**Comment by AlexJerabek** [comment]
@alison-mk, thanks for the review. The inline image should be fixed. I also took your typo fix suggestions.

**Review by alison-mk — APPROVED** [review]

**Pull request merged**